src/annotator/util/drag-handler.tsx

```diff
@@ -93,3 +97,19 @@ export class DragHandler implements Destroyable {
     this._listeners.removeAll();
   }
 }
+
+/** Suppress delivery of an event for a brief period of time. */
+function suppressEvent(
+  target: EventTarget,
+  type: string,
+  duration: number = 0,
+) {
+  const ctrl = new AbortController();
+  target.addEventListener('click', e => e.stopPropagation(), {
+    capture: true,
+    signal: ctrl.signal,
```

Comment:
Is the signal in event handlers available in all browsers we support?

Reply:
Ok, yeah. All major browsers support this since 2021 https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#browser_compatibility